spec/build/script/csharp_spec.rb

```diff
@@ -13,6 +13,7 @@
 
   describe 'configure' do
     it 'sets up package repository' do
+      data[:config][:os] = 'linux'
```

Comment:
Other languages depend on this behavior as well, such as [r](https://github.com/travis-ci/travis-build/blob/master/lib/travis/build/script/r.rb).